### DISCUSSION TITLE
What do class labels mean for graphs in UPFD?

### DISCUSSION THREAD

**Discussion by sbrito** [pyg-team/pytorch_geometric#3638] [Q&A] [answered]
For this [dataset](https://pytorch-geometric.readthedocs.io/en/latest/modules/datasets.html#torch_geometric.datasets.UPFD) I see that the graphs have a label of 0 or 1. I understand that this corresponds to whether an article is fake or real news. Which label corresponds to which? Is 0 real and is 1 fake news? Please let use know!

**Comment by rusty1s** [comment]
@YingtongDou

**Comment by YingtongDou** [comment, marked as answer]
1 represents fake and 0 represents real news.

**Reply by sbrito** [reply]
Thank you for your quick answer :)
